src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 
 import BaseChannelItem from './BaseChannelItem';
 import useChannelHook from '../../hooks/screen/useChannelHook';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `COLORS` from `../../utils/theme` is a new addition and it's used correctly to provide color constants for the background color of the channel item. 
```diff
+ import {COLORS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->